packages/react-components/react-rating-preview/src/components/RatingItem/useRatingItem.tsx

```diff
@@ -1,9 +1,12 @@
 import * as React from 'react';
 import { getIntrinsicElementProps, slot, useMergedRefs } from '@fluentui/react-utilities';
 import { useFocusWithin } from '@fluentui/react-tabster';
+import type { RatingProps } from '../Rating/Rating.types';
 import type { RatingItemProps, RatingItemState } from './RatingItem.types';
 import { useRatingItemContextValue_unstable } from '../../contexts/RatingItemContext';
 
+const defaultItemLabel: RatingProps['itemLabel'] = num => num + '';
```

Comment:
I think typing it as `RatingProps['itemLabel']` allows it to be `undefined`? Although maybe not, since it would otherwise break on line 77 where the function is called without checking for undefined. Anyways, this could instead be:
```suggestion
const defaultItemLabel: (rating: number) => rating + '';
```